apps/jobs/views.py

```diff
@@ -633,7 +633,12 @@ def update_submission(request, challenge_pk):
 
     challenge_phase_pk = request.data.get('challenge_phase')
     submission_pk = request.data.get('submission')
-    submission_status = request.data.get('submission_status').lower()
```

Comment:
Instead of adding all these lines, can we set it to default to `''` and then it will get handled easily. No need to put try catch for this.

Reply:
All the other rest fields are taken care of using `''` but this field is important as challenge host must send us the submission status once the submission is submitted by a participant. I don't think we can use `''` in this field. That's why I have added a try-excpt block.
Let me know your thoughts about it.

Reply:
If you set it to `''` as the default field then line https://github.com/Cloud-CV/EvalAI/pull/1933/files#diff-42c752186f5f6b2ed96b92afa242a6f3R650 will already take care of the invalid submission status. 

Currently, it looks like there are extra try-catch blocks which can be handled with just one existing try-catch block. Do you agree with what I am saying? Let me know if you see any concern with the approach I am suggesting.

Reply:
yeah, I agree. I didn't see that we are handling it later in the code.

Reply:
No worries. Let me know when you are done with that change. We can then merge this PR. :)

Reply:
Yup, updated it.